### PR TITLE
swap open with opn to remove critical error on audit

### PR DESCRIPTION
**Pull request by Peter-Barrett**

### CODE REVIEW

**Comment by Peter-Barrett** [comment]
@mikermcneil using the open dependency is causing a critical vulnerability according to npm audit. Changing to the opn package with the same features fixes it. 

The tests seem to be failing due to an issue with another machine dependency which I'm not too sure how to fix.

**Comment by S-Stephen** [comment]
@Peter-Barrett your changes don't appear to have added extra errors (beyond those produced via the code in master)

I attempted to make the changes against v2.0.2 however I get errors there also (although different). These errors are again are the same as running the tests included in the npm src (without having made any mods)

**Comment by Peter-Barrett** [comment]
@sgress454 will you take a look at this PR when you get a chance? Let me know if you need changes made.

File: package.json [hunk 1]
```diff
@@ -16,9 +16,10 @@
   "license": "MIT",
   "dependencies": {
     "@sailshq/lodash": "^3.10.2",
+    "ajv": "^6.5.0",
     "machine": "^15.0.0-23",
     "machinepack-json": "^2.0.0",
-    "open": "0.0.5"
+    "opn": "^5.3.0"
```
Comment (mikermcneil) [inline package.json]: @Peter-Barrett would you pin this to 5.3.0 please?

File: test/escape-as-command-line-opt.json [hunk 2]
```diff
@@ -2,7 +2,6 @@
   "machine": "escape-as-command-line-opt",
   "expectations": [
 
-    // Number
```
Comment (mikermcneil) [inline test/escape-as-command-line-opt.json]: Why the change to json rather than json5?

**Review by mikermcneil — CHANGES_REQUESTED** [review]

**Comment by mikermcneil** [comment]
@Peter-Barrett thanks for the help!  Left comments above

File: test/escape-as-command-line-opt.json (hunk 2 above)
Comment (Peter-Barrett) [reply test/escape-as-command-line-opt.json]: I was trying to figure out why the tests were failing. I'll change it back 👍

File: package.json [hunk 3]
```diff
@@ -19,7 +19,7 @@
     "ajv": "^6.5.0",
```
Comment (Peter-Barrett) [inline package.json]: should ajv be pinned too?

**Comment by Peter-Barrett** [comment]
@mikermcneil thanks for taking a look. I've made those changes.

**Review by mikermcneil — APPROVED** [review]

**Comment by mikermcneil** [comment]
@Peter-Barrett Thank you!
